packages/internal/src/storage-test-utils.js

```diff
@@ -10,6 +11,8 @@ import { bindAllMethods } from './method-tools.js';
 
 const { Fail } = assert;
 
+const trace = makeTracer('StorTU', false);
```

Comment:
this is going to add logging to lots of unrelated tests, right? hm.

Reply:
the `false` means it's disabled

Reply:
ah. yes. good morning :)